lblod/ember-rdfa-editor-lblod-plugins
# GN-4545: reset address-edit input fields when `currentAddress` changes

### Overview
This PR ensures that the `streetname`, `housenumber` and `busnumber` input fields are correctly reset when the underlying `currentAddress` property changes.

This fixes two main issues:
- When updating an address, inputting a non-existing housenumber but inserting anyway (without the housenumber), the faulty housenumber input field is not cleared
- When editing an address variable, but not updating it, and then clicking on another address variable, the inputs are now reset.

##### connected issues and PRs:
Jira: https://binnenland.atlassian.net/browse/GN-4545?atlOrigin=eyJpIjoiMzU1YmM5MDhhMjMwNGU2MDg3ZjU5Yzc1ZGRhOGY1OTAiLCJwIjoiaiJ9
#310 


### How to test/reproduce
- Insert an address variable
- Input an address in the edit-card with a faulty housenumber
- Insert the address without the housenumber
- Check if the housenumber field is correctly cleared.



### Checks PR readiness
